snippets/core/system-text-json/csharp/RoundtripImmutableStruct.cs

```diff
@@ -31,5 +28,4 @@ public static void Run()
             }
         }
     }
-   
```

Comment:
nit: I only mentioned it once, but other files have extra new lines too.

For example: https://github.com/dotnet/samples/blob/bc25d342d4cf041b953665cb5235e8d5489d34a0/snippets/core/system-text-json/csharp/SerializeRuntimePropertyExclusion.cs#L36

Consider fixing it for all the files added in https://github.com/dotnet/samples/pull/1837